homeassistant/components/netgear/sensor.py

```diff
@@ -119,3 +322,38 @@ def async_update_device(self) -> None:
         self._active = self._device["active"]
         if self._device.get(self._attribute) is not None:
             self._state = self._device[self._attribute]
+
+
+class NetgearRouterSensorEntity(NetgearRouterEntity, SensorEntity):
+    """Representation of a device connected to a Netgear router."""
+
+    _attr_entity_registry_enabled_default = False
+
+    def __init__(
+        self,
+        coordinator: DataUpdateCoordinator,
+        router: NetgearRouter,
+        entity_description: NetgearSensorEntityDescription,
+    ) -> None:
+        """Initialize a Netgear device."""
+        super().__init__(coordinator, router)
+        self.entity_description = entity_description
+        self._key = self.entity_description.key
```

Comment:
We don't need a separate attribute for this. Just use the entity description.

Reply:
done